Content.Server/Body/Systems/BodySystem.cs

```diff
@@ -40,6 +38,15 @@ private void OnRelayMoveInput(EntityUid uid, BodyComponent component, RelayMoveI
             }
         }
 
+        private void OnApplyMetabolicMultiplier(EntityUid uid, BodyComponent component, ApplyMetabolicMultiplierEvent args)
+        {
+            foreach (var (part, _) in component.Parts)
+                foreach (var mechanism in part.Mechanisms)
+                {
+                    RaiseLocalEvent(mechanism.Owner, args, false);
+                }
```

Comment:
formatting?

Reply:
I had both foreaches on the same line before, someone said to indent the second one. I don't know anymore